set.mm

```diff
@@ -61357,6 +61414,12 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       ( wf wss fss syl2anc ) ABCEHCDIBDEHFGBCDEJK $.
   $}
 
+  $( A function maps to its range iff the the range is a subset of its
+     codomain.  Generalization of ~ ffrn .  (Contributed by AV,
+     20-Sep-2024.) $)
+  ffrnb $p |- ( ran F C_ B -> ( F : A --> B <-> F : A --> ran F ) ) $=
```

Comment:
The biconditional
```
$( Characterization of a function with domain and codomain (essentially using that the
   range is always included in the codomain).
$p |- ( F : A --> B <-> ( F : A --> ran F /\ ran F C_ B ) ) $=
```
seems to offer a more precise formulation.

Reply:
I will add this theorem as ~ffrnb, and I will revise the current ~ffrnb to become ~ffrnbd in deduction form.